>yamllint -s -d '{extends: default, rules: {line-length: {allow-non-breakable-inline-mappings: true}},ignore: charts/*/templates/*.yaml}' ./.mergify.yml
./.mergify.yml
  4:3       warning  comment not indented like content  (comments-indentation)
  6:7       warning  comment not indented like content  (comments-indentation)
  8:81      error    line too long (85 > 80 characters)  (line-length)
  9:81      error    line too long (98 > 80 characters)  (line-length)
  16:7      warning  comment not indented like content  (comments-indentation)

yaml lint is failing

--- a/.mergify.yml
+++ b/.mergify.yml
@@ -1,16 +1,19 @@
 ---
 defaults:
   actions:
+    # mergify.io has removed bot_account from its free open source plan.
     comment:
-      bot_account: ceph-csi-bot
+    # bot_account: ceph-csi-bot # mergify[bot] will be commenting.
     merge:
-      bot_account: ceph-csi-bot
+      # merge_bot_account: ceph-csi-bot #mergify[bot] will be merging prs.
+      # update_bot_account: ceph-csi-bot #mergify will randomly pick and use
+      # credentials of users with write access to repo to rebase prs.
       method: rebase
       rebase_fallback: merge
       strict: smart
       strict_method: rebase
     rebase:
-      bot_account: ceph-csi-bot
+    # bot_account: ceph-csi-bot # same as update_bot_account.
 
 pull_request_rules:
   - name: remove outdated approvals

